dipy/direction/peaks.py

```diff
@@ -137,7 +137,7 @@ def peak_directions(odf, sphere, relative_peak_threshold=.5,
     elif n == 1:
         return sphere.vertices[indices], values, indices
 
-    odf_min = odf.min()
+    odf_min = np.min(odf)
```

Comment:
Haha, thank you for the test :) My goal, for once, wasn't to be faster. I changed it because this method is called from Cython and `odf` is not a np.array anymore, it's a memoryview. A memoryview doesn't have any of the numpy method (well, yes, some but now much).

**tldr** `np.func(data)` accepts np.array AND memoryview. `data.func()` doesn't.

Reply:
Oh ok ! Good to know, Thanks !